maps/MODULES/binding.lua

```diff
@@ -46,18 +66,39 @@ town_mt = {
 			return GetTownOwner(t)
 		elseif k == "faction" then
 			return GetTownFaction(t)
+		elseif k == "x" then
+			return GetTownX(t)
+		elseif k == "y" then
+			return GetTownY(t)
+		elseif k == "visitingHeroIdx" then
```

Comment:
Oh, I think I forgot that you said this. My bad. Had you not reminded me here, I don't think I would have remembered. xD